### PR TITLE
binutils: fix bootstrap build; drop noarch

### PR DESCRIPTION
**Pull request by pullmoll**
Prevent configure from finding host installed libdebuginfod.
Enable debuginfod if the chroot is ready.

Closes: #26502
Closes: #27190

### CODE REVIEW

**Review by ericonr — APPROVED** [review]
LGTM, thanks!

I will try to solve the CI stuff later, seems my installing trick wasn't fully ready yet.

**Comment by pullmoll** [comment]
I think this needs a second review because this is a core package.

**Pull request merged**
